### PR TITLE
issue 903 specs

### PR DESCRIPTION
**Pull request by nmccready**
test(center): specs for center where updates from leaflet should update the scope. issue /#903

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (nmccready) [inline .jshintrc]: @tombatossals moved the jshint settings to the .jshintrc file so editors can pick it up.

File: coffeelint.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
+{
```
Comment (nmccready) [inline coffeelint.json]: coffeelint for specs; mainly used by editors at this moment

File: test/unit/centerDirectiveSpec.coffee [hunk 3]
```diff
@@ -0,0 +1,119 @@
+'use strict'
```
Comment (nmccready) [inline test/unit/centerDirectiveSpec.coffee]: moved center specs to coffee and added leaflet to scope specs

File: test/unit/centerDirectiveSpec.coffee [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+                zoom: 8
+            centerUrlHash = undefined
+            scope.$on 'centerUrlHash', (event, u) ->
+                centerUrlHash = u
+            scope.$digest()
+            expect(centerUrlHash).toBe '9.5248:-1.8000:8'
+
+
+        it 'should update the scope.center if leaflet map is moved', (done) ->
+            element = angular.element('<leaflet center="center"></leaflet>')
```
Comment (nmccready) [inline test/unit/centerDirectiveSpec.coffee]: spec that modifies the map which should reflect on the scope

File: test/unit/centerDirectiveSpec.coffee [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+                    expect(map.getZoom()).toEqual center.zoom
+                    expect(map.getCenter().lat).toBeCloseTo 0.96658, 4
+                    expect(map.getCenter().lng).toBeCloseTo 2.02, 4
+                    map.setView L.latLng(50.5, 30.5)
+
+            @digest scope, ->
+                leafletData.getMap()
+                .then (map) ->
+                    expect(scope.center.lat).toBe 50.5
+                    expect(scope.center.lng).toBe 30.5
```
Comment (nmccready) [inline test/unit/centerDirectiveSpec.coffee]: #903

**Comment by nmccready** [comment]
@tombatossals @jessertaylor

**Comment by nmccready** [comment]
@elesdoar

**Comment by nmccready** [comment]
Merging as this is only an addition of specs

**Comment by elesdoar** [comment]
:+1:

**Pull request merged**